pringles/backends/web_display.py

```diff
@@ -0,0 +1,104 @@
+import tornado.web
+import tornado.ioloop
+from tornado.ioloop import PeriodicCallback
+import os
+from pathlib import Path
+import threading
+import asyncio
+
+from pringles.models import Model
+
+
+class ServerThread(threading.Thread):
+
+    _join_called = False
+
+    def run(self):
+        new_io_loop = asyncio.new_event_loop()
+        asyncio.set_event_loop(new_io_loop)
+        WebApplication.initialize()
+
+        ioloop = tornado.ioloop.IOLoop.current()
+
+        def healthcheck():
+            if ServerThread._join_called:
+                ioloop.stop()
+
+        # Add a healthcheck every 1 second
+        PeriodicCallback(healthcheck, 1000).start()
+
+        ioloop.start()
+        ioloop.close()
+
+    # Overriding join method to mark the ioloop to be disposed
+    def join(self):
+        ServerThread._join_called = True
+        super().join()
+
+
+web_model_display_thread = ServerThread()
```

Comment:
maybe move this down closer to the ipython_inline_display method ? Here it's just lost between two classes

Reply:
Agree.